tests/tests/bitasset_tests.cpp

```diff
@@ -928,26 +928,22 @@ BOOST_AUTO_TEST_CASE( hf_935_test )
    generate_blocks( db.get_dynamic_global_properties().next_maintenance_time, true, skip );
    generate_block( skip );
 
-   for( int i = 0; i < 8; ++i )
+   for( int i = 0; i < 6; ++i )
    {
       idump( (i) );
       int blocks = 0;
       auto mi = db.get_global_properties().parameters.maintenance_interval;
 
-      if( i == 2 ) // go beyond hard fork 890
+      if( i == 2 ) // go beyond hard fork 890, 343, 935
       {
          generate_blocks( HARDFORK_CORE_868_890_TIME - mi, true, skip );
-         generate_blocks( db.get_dynamic_global_properties().next_maintenance_time, true, skip );
       }
-      else if( i == 4 ) // go beyond hard fork 935
+      else if( i == 4 ) // go beyond hard fork 1270
       {
-         generate_blocks( HARDFORK_CORE_935_TIME - mi, true, skip );
-         generate_blocks( db.get_dynamic_global_properties().next_maintenance_time, true, skip );
-      }
-      else if( i == 6 ) // go beyond hard fork 1270
-      {
-         generate_blocks( HARDFORK_CORE_1270_TIME, true, skip );
+         generate_blocks( HARDFORK_CORE_1270_TIME - mi, true, skip );
       }
+      generate_blocks( db.get_dynamic_global_properties().next_maintenance_time, true, skip );
```

Comment:
Why this change? Only need to generate blocks to next maintenance time if `i` is in `{2,4,6}`.